### PR TITLE
Removes the transition for the sake of nesting

### PR DESCRIPTION
**Pull request by abea**
Needs the designer blessing.

Our transition group put a `span` in between the draggable component and the target list. This looks to have got in the way when the list was empty. No one is happy, the sign if a good compromise.

### CODE REVIEW

File: modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue [hunk 1]
```diff
@@ -7,90 +7,86 @@
...
-          <!-- {{ options.startCollapsed }} -->
-          <button
-            v-if="row.children && row.children.length > 0"
-            class="apos-tree__row__toggle" data-apos-tree-toggle
-            aria-label="Toggle section" :aria-expanded="!options.startCollapsed"
-            @click="toggleSection($event)"
-          >
-            <chevron-down-icon :size="16" class="apos-tree__row__toggle-icon" />
-          </button>
+    <li
```
Comment (abea) [inline modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue:10]: Hide whitespace changes.

File: modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue [hunk 2]
```diff
@@ -7,90 +7,86 @@
...
-              v-if="col.icon" :is="icons[col.icon]"
-              class="apos-tree__cell__icon"
-            />
-            <span v-show="!col.iconOnly">
-              {{ row[col.name] }}
-            </span>
-          </component>
-        </div>
-        <AposTreeRows
-          v-if="row.children"
```
Comment (abea) [inline modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue:71]: This does nothing. `children` is always an empty array at least.

**Review by stuartromanek — APPROVED** [review]
😚 👌 💥

**Comment by abea** [comment]
... I think there may be a way to bring it back by setting it to be a div.

**Pull request merged**